.yamllint

```diff
@@ -0,0 +1,10 @@
+# vi: ft=yaml
```

Comment:
Ok. Can we add it to style check?

Reply:
Sure, but there will lots of failures right now.
But agree, let's drop `.yamllint` and add it along with style check.